src/apps/material/helper.ts

```diff
@@ -543,19 +552,18 @@ export const getAvailability = async ({
 export const useGetHoldings = ({
   faustIds,
   config,
-  useAvailabilityBlacklist = false,
+  blacklist,
   options
 }: {
   faustIds: FaustId[];
   config: UseConfigFunction;
-  useAvailabilityBlacklist?: boolean;
+  blacklist: BlacklistType;
   options?: {
     query?: UseQueryOptions<Awaited<ReturnType<typeof getHoldingsV3>>>;
   };
 }) => {
-  const blacklistedBranches = useAvailabilityBlacklist ? "both" : "pickup";
   const { data, isLoading, isError } = useGetHoldingsV3(
-    getBlacklistedQueryArgs(faustIds, config, blacklistedBranches),
+    getBlacklistedQueryArgs(faustIds, config, blacklist),
```

Comment:
:+1: This change makes sense to me.

We could consider using the same default value as was used before. I think it is OK to make it explicit though.